src/node/utils/tar.json

```diff
@@ -82,6 +82,4 @@
   , "pluginfw/shared.js"
   , "pluginfw/hooks.js"
   ]
-, "jquery.js": ["jquery.js"]
```

Comment:
I coulda swore this line was necessary to support plugins that have frontend code that does `require('jquery')`, but maybe not.

Reply:
I would expect `require('ep_etherpad-lite/static/js/jquery')` in this case, which will work and `require('jquery')` to resolve to a local dependency of the plugin.